src/Enterspeed.Source.SitecoreCms.V8/Services/EnterspeedConfigurationService.cs

```diff
@@ -182,15 +182,28 @@ private bool HasNoConfigurationSetUp(Item enterspeedConfigurationItem)
 
         private bool IsConfigurationUpdated(Item item, out DateTime currentUpdatedDate)
         {
-            currentUpdatedDate = item.Children.Max(i => i.Statistics.Updated);
+            var database = _factory.GetDatabase("web");
+            var languages = _languageManager.GetLanguages(_factory.GetDatabase("web"));
```

Comment:
Do we need to fetch the web database again here, we just fetched it in the line above?